src/lib/sw_engine/tvgSwRenderer.cpp

```diff
@@ -594,10 +595,10 @@ void* SwRenderer::prepareCommon(SwTask* task, const RenderTransform* transform,
     task->surface = surface;
     task->mpool = mpool;
     task->flags = flags;
-    task->bbox.min.x = max(static_cast<SwCoord>(0), static_cast<SwCoord>(vport.x));
-    task->bbox.min.y = max(static_cast<SwCoord>(0), static_cast<SwCoord>(vport.y));
-    task->bbox.max.x = min(static_cast<SwCoord>(surface->w), static_cast<SwCoord>(vport.x + vport.w));
-    task->bbox.max.y = min(static_cast<SwCoord>(surface->h), static_cast<SwCoord>(vport.y + vport.h));
+    task->bbox.min.x = std::max(static_cast<SwCoord>(0), static_cast<SwCoord>(vport.x));
```

Comment:
We don't much prefer to use stl stuff for sake of less binary size, 

Could you please add the macros in tvgMath.h
```
#define mathMin(x, y) (((x) > (y)) ? (x) : (y))
#define mathMax(x, y) (((x) < (y)) ? (x) : (y))
```

then place with it?

Reply:
I definitely agree with that, will do.

Reply:
Done (with signs fixed as you had mixed up min and max ;)).